tests/components/miele/test_sensor.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for miele sensor module."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_sensor_states(
+    hass: HomeAssistant,
+    mock_miele_client: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    snapshot: SnapshotAssertion,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test sensor state."""
+
+    with patch("homeassistant.components.miele.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
```

Comment:
don't we have a really nice fixture for this?

Reply:
I have looked around without success. Can you point me in some direction?